app/models/game.rb

```diff
@@ -0,0 +1,12 @@
+class Game < ActiveRecord::Base
+  validates :name, presence: true
+  validates :white_player_id, presence: true
+  validates :black_player_id, presence: true
```

Comment:
We will run into issues down the road with these being required due to needing to join a game. There's not going to be many instances in which a game will be created immediately with both players. Maybe just a required on whichever the first player is?